### PR TITLE
feat: allow custom loading manager to useTexture

### PR DESCRIPTION
**Pull request by hawk86104**
fiexd TextureLoader use THREE.DefaultLoadingManager as default loading manager form: #432

### CODE REVIEW

File: src/composables/useTexture/index.ts [hunk 1]
```diff
@@ -114,9 +114,10 @@ export async function useTexture<TextureMap extends PBRUseTextureMap>(
 
 export async function useTexture(
   paths: readonly [string] | string[] | PBRUseTextureMap,
+  manager?: LoadingManager,
 ): Promise<Texture | Texture[] | PBRTextureMaps> {
-  const loadingManager = new LoadingManager()
-  const textureLoader = new TextureLoader(loadingManager)
+  // fiexd TextureLoader use THREE.DefaultLoadingManager as default loading manager
```
Comment (alvarosabu) [inline src/composables/useTexture/index.ts:119]:
```suggestion
  
```

Comment (alvarosabu) [reply src/composables/useTexture/index.ts:119]: @hawk86104 you can remove the comment is clear with the code.

**Review by alvarosabu — APPROVED** [review]

**Review by alvarosabu — CHANGES_REQUESTED** [review]
Thanks @hawk86104 for this PR, could you also update the documentation under `docs/` to reflect these changes?

These changes should be a `feat` instead of a `chore` or `fix` since they add a new option to the end user.

**Comment by andretchen0** [comment]
@hawk86104 

Nice addition. Small and flexible!

**Comment by hawk86104** [comment]
> Thanks @hawk86104 for this PR, could you also update the documentation under `docs/` to reflect these changes?
> 
> These changes should be a `feat` instead of a `chore` or `fix` since they add a new option to the end user.

I have just submitted the update, along with the corresponding API documentation.

File: src/composables/useTexture/index.ts [hunk 2]
```diff
@@ -114,9 +114,10 @@ export async function useTexture<TextureMap extends PBRUseTextureMap>(
 
 export async function useTexture(
   paths: readonly [string] | string[] | PBRUseTextureMap,
+  manager?: LoadingManager,
```
Comment (alvarosabu) [inline src/composables/useTexture/index.ts:117]: @hawk86104 the default value doesn't need to be instantiated like manager: `new LoadingManager()` ?

**Review by alvarosabu — CHANGES_REQUESTED** [review]

File: src/composables/useTexture/index.ts (hunk 2 above)
Comment (hawk86104) [reply src/composables/useTexture/index.ts:117]: https://threejs.org/docs/index.html#api/en/loaders/TextureLoader

Comment (hawk86104) [reply src/composables/useTexture/index.ts:117]: The three.js documentation already states that default is [THREE.DefaultLoadingManager](https://threejs.org/docs/index.html#api/en/loaders/managers/LoadingManager)

Comment (hawk86104) [reply src/composables/useTexture/index.ts:117]: I have also tested it in my own project, and there were no issues.

**Review by alvarosabu — APPROVED** [review]

**Pull request merged**